src/Utils/parserCarList.js

```diff
@@ -0,0 +1,13 @@
+import { PATTERN } from "../Constants/constant.js";
+
+/**
+ *
+ * @param {string} carList
+ * @returns string[]
+ * @summary 입력받은 자동차 리스트를 쉼표로 구분하여 배열로 만든다.
+ */
+const parserCarList = (carList) => {
+  const carListArr = carList.split(PATTERN.COMMA);
+  return carListArr;
```

Comment:
```suggestion
  return carList.split(PATTERN.COMMA);
```
이렇게 생략해도 무방할 것 같아요~!

Reply:
아 그렇네요!! `carListArr`라는 변수가 사용되질 않아서 바로 return 하는 방식이 더 좋은 것 같아요. 
피드백 감사합니다~~!